docs/Using-Fleet/Adding-hosts.md

```diff
@@ -279,7 +281,20 @@ expiration setting. To configure this setting, in the Fleet UI, head to **Settin
 
 ## Add Chromebooks with the Fleetd Chrome extension
 
-> Documentation for adding hosts on ChromeOS is coming soon. In the mean time, basic instructions are available in the [fleetd-chrome README](https://github.com/fleetdm/fleet/blob/main/ee/fleetd-chrome/README.md).
+Visit the Google Admin console. In the navigation menu, visit Devices > Chrome > Apps & Extensions > Users & browsers.
+
+Select the relevant organizational unit, users, or group where you want the fleetd Chrome extension to be installed.
+
+In the bottom right, click the yellow "+" button and select "Add Chrome app or extension by ID."
+
+Visit your Fleet instance and select Hosts > Add Hosts and select ChromeOS in the popup modal.
+
+Enter the Extension ID, custom URL and Policy for extensions using the data provided in the modal.
```

Comment:
Rename custom URL to Installation URL to match field name in google admin UI

Reply:
```suggestion
Enter the "Extension ID," "Installation URL," and "Policy for extensions" using the data provided in the modal.
```